Volume/Models/WeeklyDebrief.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  WeeklyDebrief.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 11/17/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import Apollo
+import Foundation
+
+struct WeeklyDebrief {
+    var creationDate: Date
+    var expirationDate: Date
+    var numShoutouts: Int
+    var numReadArticles: Int
+    var numBookmarkedArticles: Int
+    var readArticles: [Article]
+    var randomArticles: [Article]
+    
+    init(from weeklyDebrief: WeeklyDebriefFields) {
+        /*
+         schema changes required:
+         - createdAt -> creationDate
+         - numShoutouts, numReadArticles, numBookmarkedArticles -> type Int
+         */
```

Comment:
Why is this here? You shouldn't commit a partial feature to main. If it is a work in progress, then make it a draft and don't merge until everything is done.